### PR TITLE
Fix #1481: Removed loading when empty upload

### LINKED ISSUE #1481: Create empty challenge
- When a user goes to the create a challenge and clicks on the upload button without adding anything, the loading bar gets stuck on the screen
screenshot from 2018-01-20 21-14-00

I would like to take this issue.

### PR DESCRIPTION
**Pull request by viditjain08**
Fix #1481.

- Stopped the loader when the file upload was empty.

### CODE REVIEW

File: requirements/common.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@ djangorestframework-expiring-authtoken==0.1.4
 django-cors-headers==1.3.1
 django-rest-auth[with_social]
 drfdocs==0.0.11
+pathlib==1.0.1
```
Comment (aka-jain) [inline requirements/common.txt]: Why you have added this? @viditjain08

Comment (viditjain08) [reply requirements/common.txt]: Travis Build was failing because there was no module named pathlib, and it's failing now as well

**Comment by aka-jain** [comment]
@RishabhJain2018 can you please look into this.

**Comment by viditjain08** [comment]
Sorry, I added the same changes as deshraj as I couldn't help but correct the build failed for the pr

**Comment by deshraj** [comment]
Ah, sorry @viditjain08. I am really sorry. I missed your pull request. I should have looked at this before working on it.

**Comment by deshraj** [comment]
I think your pull request is still valid since it solves one issue.

**Comment by viditjain08** [comment]
Thanks @deshraj

**Comment by deshraj** [comment]
@viditjain08: I am not sure which file upload is this? Is this the one where we try to  upload the challenge or the one where a participant tries to make a submission?

**Comment by viditjain08** [comment]
This is the challenge file upload

**Comment by deshraj** [comment]
LGTM. Thanks!

**Pull request merged**